### PR TITLE
feat: GEO-888 Schedule to email about expiring announcements

### PR DESCRIPTION
**Pull request by jer3k**
https://finrms.atlassian.net/browse/GEO-888

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://pay-transparency-pr-702-frontend.apps.silver.devops.gov.bc.ca)
- [Admin Frontend](https://pay-transparency-pr-702-admin-frontend.apps.silver.devops.gov.bc.ca)
- [Backend external API console](https://pay-transparency-pr-702-backend-external.apps.silver.devops.gov.bc.ca/api/V1/docs)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/fin-pay-transparency/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/fin-pay-transparency/actions/workflows/merge.yml)

### CODE REVIEW

File: .vscode/settings.json
Comment (goemen) [inline .vscode/settings.json:1]: Are this changes necessary

File: backend/src/v1/services/sso-service.ts [hunk 1]
```diff
@@ -172,22 +188,9 @@ export class SSO {
...
-            (x) => x.preferred_username == user.preferredUserName,
-          ).admin_user_id,
-          displayName: user.displayName,
-          effectiveRole: user.roles.includes(PTRT_ADMIN_ROLE_NAME)
-            ? PTRT_ADMIN_ROLE_NAME
-            : PTRT_USER_ROLE_NAME,
-          roles: user.roles,
-        },
-    );
+    console.log(JSON.stringify(localUsers, null, 2));
```
Comment (goemen) [inline backend/src/v1/services/sso-service.ts:191]: Remove the log

File: .vscode/settings.json
Comment (jer3k) [reply .vscode/settings.json:1]: Yes. We are required to follow the rules of eslint as best as we can. Somehow it got disabled, so it needs to be enabled.
The "prettier" setting is because by default the prettier extension will use it's default rules if there are no config settings anywhere. Unfortunately prettier messes up some of the yml files in the "charts" folder. I can't remember which one I was looking at though though. So if there's no config for prettier, I think it's best not to run it.

File: backend/src/v1/services/sso-service.ts (hunk 1 above)
Comment (jer3k) [reply backend/src/v1/services/sso-service.ts]: Thanks

File: .vscode/settings.json
Comment (jer3k) [reply .vscode/settings.json:1]: I found it, prettier messes up this file charts\fin-pay-transparency\templates\configmap.yaml

**Review by goemen — APPROVED** [review]
